include/ck/utility/workgroup_synchronization.hpp

```diff
@@ -0,0 +1,68 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+#include "ck/host_utility/hip_check_error.hpp"
+
+namespace ck {
+
+// Initialization flag of Barrier object, can be any value except for zero
+static constexpr int BarrierInitFlag = 0x7856;
+
+// only the first thread-block in the synchronizaton group is supposed to call this function
+static __device__ void gms_init(int NumWarps, int* p_control_bits)
```

Comment:
Add comment to warn user to make sure p_control_bits is zero before gms_init

Reply:
https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/771/commits/67e19bcfb374ad2ebe722f0868b2b83e4c0ca0f0